contrib/kind.sh

```diff
@@ -20,13 +20,16 @@ run_kubectl() {
   done
 }
 
+MNODE_COUNT=1
```

Comment:
Can you call it `MASTER_COUNT`? I just got rid of all the remaining references to "master nodes"

Reply:
sure can rename MNODE_COUNT to MASTER_COUNT, will fix in the next commit